README.md

```diff
@@ -54,28 +54,14 @@ To deploy to Heroku you will need to add the environment variables _manually_ (_
 > see: https://translate.google.com/#pt/en/tudo
 
 ###Front-end
-**20150720:** @msmichellegar, @iteles and Anneka S brainstormed on some front-end visuals and created some simple wireframes:
-https://docs.google.com/presentation/d/11JJjfQ-MtnfnRGQT9KREmqUHQj4wkgB7tjSxXhXyNJg/edit#slide=id.p
 
-<img width="918" alt="proposed-initial-stripped-down-design" src="https://cloud.githubusercontent.com/assets/4185328/8803847/24905988-2fc1-11e5-8a10-754ab36236ad.png">
+@msmichellegar and @iteles have styled three screens for the project: the "login"/landing page, the home feed, and an issue view. Original wireframes for these screens (and an archive of discussed designs) can be found [here](https://docs.google.com/presentation/d/11JJjfQ-MtnfnRGQT9KREmqUHQj4wkgB7tjSxXhXyNJg/edit#slide=id.p).
 
-#####Challenges:
-+ What is the ideal user flow throughout the app? How do typical people (technical and non-technical) find the issues they're looking for?  
-+ How do we ensure that issues are _not_ buried **more than 1 click deep**?
+We have worked to a very stripped-down design for the MVP, and will be adding more functionality as agreed on by the DWYL community (customisation, organisation and repo views, toggle views).
```

Comment:
Hey @msmichellegar thanks for this. Next time please use lowercase for dwyl as that is more inline with the current branding. 

I will fix this for you now.